apps/example/src/frontend/src/components/header.tsx

```
@@ -0,0 +1,27 @@
+    <div className="flex px-16 py-4 border-b border-neutral-300 gap-2.5 justify-between items-center">
+      <div className="flex gap-16 items-center">
+        <div className="flex gap-2 items-center justify-center">
+          <img src="/logo.png" alt="Curate.fun Logo" className="h-8 w-8 " />
+          <h1 className="text-3xl leading-7">Curate.fun Plugins</h1>
+        </div>
+        <div className="flex items-center justify-center gap-3 ">
+          {navigation.map((link) => (
+            <a
+              href={link.href}
+              className="text-sm font-medium leading-6 min-w-[64px]"
+            >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)